README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/phac-nml/irida.svg?branch=development)](https://travis-ci.org/phac-nml/irida)
```

Comment:
I'm noticing on your fork this is showing "Build: Unknown" even though the tests passed.  Is this the right format?  Or is it because you're linking to the `phac-nml` repo and it's not merged yet?

Reply:
Thanks.  Yep, it's the right format.  It shows `unknown` because it's linked up the the development branch in the right now in the `phac-nml` repo which has no tests run on it yet.